### ISSUE TITLE
install-snapshot: Comma separated "repo" is not parsed properly

### ISSUE THREAD

**Issue by StevenMassaro** [liquibase/liquibase-sdk-maven-plugin#1]
I am trying to install builds from OSS liquibase and liquibase-pro. I followed the instructions in the readme and set up the plugin like this:

```
      - name: Install Snapshot Liquibase
        env:
          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
        run: |
          mvn -B versions:set-property -Dproperty=liquibase-core.version -DnewVersion=0-SNAPSHOT
          mvn -B liquibase-sdk:install-snapshot \
            "-Dliquibase.sdk.branchSearch=${{ steps.extract_branch.outputs.branch }},master" \
            "-Dliquibase.sdk.repo=liquibase/liquibase,liquibase/liquibase-pro"
          buildOverview="$(mvn -B -q liquibase-sdk:get-build-info '-Dliquibase.sdk.buildInfo.outputKey=overview')"
          echo "::notice :: Installed Snapshot Liquibase $buildOverview"
```

but I get this error:

```
[INFO] Looking for s3-integration-tests,master from a run in liquibase,liquibase-pro
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.725 s
[INFO] Finished at: 2022-08-03T12:34:12Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.liquibase.ext:liquibase-sdk-maven-plugin:0.9.1:install-snapshot (default-cli) on project s3-remote-accessor-extension: https://api.github.com/repos/liquibase/liquibase,liquibase-pro {"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/repos#get-a-repository"} -> [Help 1]
```

Configuring the plugin with the repo owner's name also causes the same error:

```
"-Dliquibase.sdk.repo=liquibase,liquibase-pro"
```

**Comment by StevenMassaro** [comment]
This is probably because the latest code hasn't been released yet.

**Comment by StevenMassaro** [comment]
Confirmed, this works as desired in db59d146e58a1dc0bf8e38182b306150ea63c4c7

**Comment by nvoxland** [comment]
This should be fixed in #3
